src/index.js

```diff
@@ -1,3 +1,4 @@
+import Vue from 'vue';
```

Comment:
Rollup outputs two new warnings:

```
(!) Unresolved dependencies
(!) Missing global variable name
```

Please resolve those. Thx!

Reply:
vue has been added as a dependency in package.json